docs/promotions/index.rst

```diff
@@ -0,0 +1,28 @@
+.. _prmotions-index:
+
+========================
+CrateDB Cloud promotions
+========================
+
+Crate.io regularly offers promotions, consisting of either free or discounted
+offers on CrateDB Cloud database-as-a-service products. Promotions can involve
+discounts of up to 100% on any dimensions of the product: for example, there
+can be promotions on specific `subscription plans`_ or on `scale units`_ (the
+number of nodes in a cluster) within a plan. Promotions may last indefinitely
+or for limited periods of time.
+
+Another promotional mechanism that should be useful for testing purposes or for
+users new to CrateDB Cloud is our Free trial budget system. This provides you
+with a free credit whenever you create a new organization. You can spend this
+credit on any CrateDB Cloud development plan you choose.
```

Comment:
Again, would not mention that this is given every time you create a new org ;-) Maybe something like:

```suggestion
CrateDB Cloud also offers new users $200 in free credits upon signup. You can spend these
credits on any CrateDB Cloud cluster configuration you choose.
```

Reply:
Ah, that's probably true. Better to obfuscate that a little bit. Better now?